FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -651,3 +684,42 @@ def export_storeHostInfo( self ):
       return S_ERROR( result[ 'Message' ] )
 
     return S_OK( 'Profiling information logged correctly' )
+
+  types_storeProfiling = []
+  def export_storeProfiling( self ):
+    """
+    Retrieves and stores into ElasticSearch profiling information about the components on the host
+    """
+    result = ComponentInstaller.getStartupComponentStatus( [] )
+    if not result[ 'OK' ]:
+      gLogger.error( result[ 'Message' ] )
+      return S_ERROR( result[ 'Message' ] )
+    startupComps = result[ 'Value' ]
+
+    result = ComponentInstaller.getSetupComponents()
+    if not result[ 'OK' ]:
+      gLogger.error( result[ 'Message' ] )
+      return S_ERROR( result[ 'Message' ] )
+    setupComps = result[ 'Value' ]
+
+    # Get the profiling information for every running component and send it to RabbitMQ
+    for cType in setupComps:
+      for system in setupComps[ cType ]:
+        for comp in setupComps[ cType ][ system ]:
+          pid = startupComps[ '%s_%s' % ( system, comp ) ][ 'PID' ]
+          profiler = Profiler.Profiler( pid )
```

Comment:
getStartupComponentStatus also return CPU, Memory,... usage. Profiler much better maybe we can drop that functionality in getStartupComponentStatus method and use profiler here as well.